tests/multus_connectivity_test.go

```diff
@@ -0,0 +1,171 @@
+			for _, nodeToIp := range nodesToIpList {
+				if nodeToIp.name != nodeName {
+					nextNodeIp = nodeToIp.ip
+					break
+				}
+			}
+			tests.WaitUntilResourceReadyByName(
+				"pod",
+				pod.Name,
+				ktests.KubeVirtInstallNamespace,
+				"-o=jsonpath='{.status.phase}'",
+				"Running",
+				)
+
+			_, err = ktests.ExecuteCommandOnPod(
+				virtClient, &pod, podContainer, []string{ovsVsCtl, "add-br", bridgeName},
+				)
+			Expect(err).ToNot(HaveOccurred())
+			_, err = ktests.ExecuteCommandOnPod(
+				virtClient, &pod, podContainer, []string{
+					ovsVsCtl,
+					"add-port",
+					bridgeName,
+					"vxlan",
+					"--",
+					"set", "Interface", "vxlan", "type=vxlan", "options:remote_ip="+nextNodeIp,
+				},
+			)
+			Expect(err).ToNot(HaveOccurred())
+		}
+
+		for i := range vmiList{
+			vmiList[i] = ktests.NewRandomVMIWithEphemeralDisk(ktests.RegistryDiskFor(ktests.RegistryDiskAlpine))
+			vmiList[i].Spec.Domain.Devices.Interfaces = []v1.Interface{
+				{
+					Name: "default",
+					InterfaceBindingMethod: v1.InterfaceBindingMethod{
+						Bridge: &v1.InterfaceBridge{},
+					},
+				},
+				{
+					Name: "vm-ovs-vlan-net",
+					InterfaceBindingMethod: v1.InterfaceBindingMethod{
+						Bridge: &v1.InterfaceBridge{},
+					},
+				},
+			}
+			vmiList[i].Spec.Networks = []v1.Network{
+				{
+					Name: "default",
+					NetworkSource: v1.NetworkSource{
+						Pod: &v1.PodNetwork{},
+					},
+				},
+				{
+					Name: "vm-ovs-vlan-net",
+					NetworkSource: v1.NetworkSource{
+						Multus: &v1.MultusNetwork{
+							NetworkName: "ovs-vlan-net",
+						},
+					},
+				},
+			}
+			vmiList[i], err = virtClient.VirtualMachineInstance(tests.NamespaceTestDefault).Create(vmiList[i])
+			Expect(err).ToNot(HaveOccurred())
+			ktests.WaitForSuccessfulVMIStart(vmiList[i])
+			expecter, err := ktests.LoggedInAlpineExpecter(vmiList[i])
+			Expect(err).ToNot(HaveOccurred())
+			defer expecter.Close()
+
+			_, err = expecter.ExpectBatch([]expect.Batcher{
+				&expect.BSnd{S: fmt.Sprintf("ip add a "+vmsIp[i]+"/24 dev eth1\n")},
+				&expect.BExp{R: ""},
+				&expect.BSnd{S: fmt.Sprintf("ip link set up eth1\n")},
+				&expect.BExp{R: ""},
+			}, 60)
+			Expect(err).ToNot(HaveOccurred())
+		}
+	})
+
+	AfterEach(func() {
+		for _, pod := range pods.Items {
+			_, err = ktests.ExecuteCommandOnPod(
+				virtClient, &pod, pod.Spec.Containers[0].Name, []string{ovsVsCtl, "del-br", bridgeName},
+			)
+			Expect(err).ToNot(HaveOccurred())
+		}
+		tests.DeleteResourceWithFilePath(ovsPodYaml)
+		tests.DeleteResourceWithFilePath(ovsVlanNet)
+	})
+
+	Context("Connectivity between VMs over OVS interfaces", func() {
+		It("Two VMs ping each other should be successful", func() {
+			for i := range vmiList{
+				err := ktests.CheckForTextExpecter(vmiList[i], []expect.Batcher{
+					&expect.BSnd{S: fmt.Sprintf("ping -w 3 %s \n", vmsIp[1-i])},
+					&expect.BExp{R: "3 packets received"},
+				}, 60)
+				Expect(err).ToNot(HaveOccurred())
+			}
+		})
+	})
+})
```

Comment:
Looks good!!! How about adding some test for the connectivity of the vms' default network to make sure the it won't be affected by the extra network?